js/network.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+window.network = (function () {
+
+  return {
+    errorHandler: function (message) {
+      console.log('Server error: ', message);
+    },
+
+    successHandler: function (message) {
+      console.log('Server responce: ', message);
+    },
+
+    receive: function (onLoad) {
+      var reciveDataUrl = 'https://js.dump.academy/keksobooking/data';
+      var xhrRecive = new XMLHttpRequest();
+
+      function xhrReciveHandler(event) {
+        if (event.target.status === 200) {
+          onLoad(event.target.responseText);
+        } else {
+          window.network.errorHandler(event.target.status);
+        }
+      }
+
+      function xhrErrorHandler(event) {
+        window.network.errorHandler(event.target.status);
+      }
+
+      function xhrTimeoutHandler() {
+        console.log('Connection time out');
+      }
+
+      xhrRecive.addEventListener('load', xhrReciveHandler);
+      xhrRecive.addEventListener('error', xhrErrorHandler);
+      xhrRecive.addEventListener('timeout', xhrTimeoutHandler);
+
+      xhrRecive.timeout = 10000;
+      xhrRecive.open('GET', reciveDataUrl);
+      xhrRecive.send();
+    },
+
+    send: function (data) {
```

Comment:
should accept two more arguments onSuccess handler and onError handler

Reply:
done